content/en/docs/concepts/cluster-administration/monitoring.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Monitoring Control Plane Components
+reviewers:
+- brancz
+- logicalhan
+- RainbowMango
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+
+System component metrics can give a better look into what is happening inside them. Metrics are particularly useful for building dashboards and alerts.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Metrics in Kubernetes
+
+Kubernetes control plane components use prometheus as the default client for exposing metrics. In most cases those metrics are available on `/metrics` endpoint of the HTTP server. For components that doesn't expose endpoint by default it can be enabled using `--bind-address` flag.
+
+Examples of those components:
+* kube-controller-manager
+* kube-proxy
+* kube-apiserver
+* kube-scheduler
+* kubelet (metrics exposed on `/metrics/cadvisor` and `/metrics/resource` do not have same guarantee)
+
+For clusters with RBAC enabled accessing metrics requires authorization through service account with `ClusterRole` for `/metrics` url.
+
+```
+apiVersion: rbac.authorization.k8s.io/v1	
+kind: ClusterRole	
+metadata:	
+  name: prometheus	
+rules:	
+  - nonResourceURLs:	
+      - "/metrics"	
+    verbs:	
+      - get
+```
+
+## Metric Lifecycle 
+
+Alpha metric -> Stable metric -> Deprecated metric -> Hidden metric -> Deletion
+
+Alpha metrics have no stability guarantees; as such they can be modified or deleted at any time.
+
+Stable metrics can be guaranteed to not change, except that the metric may become marked deprecated for a future Kubernetes version. By not change, we mean three things:
```

Comment:
```suggestion
Stable metrics can be guaranteed to not change, except that the metric may become marked deprecated for a future Kubernetes version. Specifically, stability means:
```
(from [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we))